index.rst

```diff
@@ -319,13 +309,11 @@ PyTorch in Other Languages
    :hidden:
    :caption: Text
 
-   beginner/chatbot_tutorial
    intermediate/char_rnn_classification_tutorial
    intermediate/char_rnn_generation_tutorial
-   beginner/deep_learning_nlp_tutorial
    intermediate/seq2seq_translation_tutorial
-   beginner/translation_torchtext_tutorial
```

Comment:
why we remove those tutorials from the website?

Reply:
Great question. I want our Text tutorials to be clean and easy-to-follow. The two tutorials I'm deleting here get in the way of that:

1. The chatbot tutorial is very long and has multiple known issues.
2. The Deep Learning with NLP tutorials cover material outside of the main sequence of tutorials that actually show the key points of using PyTorch for Text. 

More generally, this PR is part of a broader effort to clean up the tutorials I've shared with Joe Spisak; the details, that Joe and others have reviewed and agreed to, are in [this Google Sheet](https://docs.google.com/spreadsheets/d/1vlaKU5uMDbyZvlDoKyJUFt2vnm94rlR9JuDg8hbLv4w).

Reply:
Great question. I want our Text tutorials to be clean and easy-to-follow. The two tutorials I'm deleting here get in the way of that:

1. The chatbot tutorial is very long and has multiple known issues.
2. The Deep Learning with NLP tutorials cover material outside of the main sequence of tutorials that actually show the key points of using PyTorch for Text.

More generally, this PR is part of a broader effort to clean up the tutorials I've shared with Joe Spisak; the details, that Joe and others have reviewed and agreed to, are in [this Google Sheet](https://docs.google.com/spreadsheets/d/1vlaKU5uMDbyZvlDoKyJUFt2vnm94rlR9JuDg8hbLv4w).